src/Interpreters/executeQuery.cpp

```diff
@@ -721,22 +722,6 @@ BlockIO executeQuery(
     return streams;
 }
 
-BlockIO executeQuery(
-        const String & query,
-        Context & context,
-        bool internal,
-        QueryProcessingStage::Enum stage,
-        bool may_have_embedded_data,
-        bool allow_processors)
-{
-    BlockIO res = executeQuery(query, context, internal, stage, may_have_embedded_data);
-
-    if (!allow_processors && res.pipeline.initialized())
-        res.in = res.getInputStream();
-
-    return res;
-}
-
```

Comment:
I think we cannot remove it because of backward compatibility.
Tests should fail

Reply:
But there is no callsite of this function.

Reply:
There is in code used by side projects.

Reply:
It's better to fix side projects than to keep unused garbage in our codebase.